server/src/main/java/org/elasticsearch/index/mapper/TextFieldMapper.java

```diff
@@ -129,7 +130,7 @@
         private double fielddataMinFreq = Defaults.FIELDDATA_MIN_FREQUENCY;
         private double fielddataMaxFreq = Defaults.FIELDDATA_MAX_FREQUENCY;
         private int fielddataMinSegSize = Defaults.FIELDDATA_MIN_SEGMENT_SIZE;
-        private SimilarityProvider similarity;
+        protected SimilarityProvider similarity;
```

Comment:
I generally feat `protected` mutable variables. What's up with this?

Reply:
I get that this is the builder, but do we plan to extend it or something? Everything else manages to be `private` which makes me less nervous.

Reply:
It's extended by the `AnnotatedTextFieldMapper` and used there.